modules/aws/bootstrap/README.md

```diff
@@ -0,0 +1,55 @@
+# Bootstrap Module
+
+This [Terraform][] [module][] manages [AWS][] resources only needed during cluster bootstrapping.
+It uses [implicit provider inheritance][implicit-provider-inheritance] to access the [AWS provider][AWS-provider].
+
+## Example
+
+Set up a `main.tf` with:
+
+```hcl
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "ami" {
+  source = "/github.com/openshift/installer//modules/aws/ami"
```

Comment:
//

Reply:
> //

It's intentional (docs [here][1]).  I'll push a new commit message linking those docs as well.

[1]: https://www.terraform.io/docs/modules/sources.html#modules-in-package-sub-directories